AmazingMeSample/app/src/main/java/amazingme/database/FirebaseHelper.java

```diff
@@ -0,0 +1,86 @@
+package amazingme.database;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AlertDialog;
+import android.util.Log;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+
+import amazingme.activities.app.MainMenu;
+import amazingme.activities.app.RegisterActivity;
+
+public class FirebaseHelper {
+
+    //gonna keep this so we can more easily communicate with the firebase
+
+    private FirebaseDatabase database = FirebaseDatabase.getInstance();
+    private DatabaseReference reference = database.getReference("message");
+
+    public FirebaseHelper(){
+        System.out.println("HERE WE GO");
```

Comment:
do we wanna log here? anything that needs to be initialized? if not we can just keep it empty. my sucky SOP lines aren't helpful :)

Reply:
I'll just get rid of them